less/pages/thunderbird.less

```diff
@@ -22,9 +26,44 @@
     }
   }
   .submit-btn {
-    background-color: #3D99F9 !important;
+    background-color: #3D99F9;
+    &:hover {
+      background-color: #53A0F0;
+    }
+  }
+  .payment-type + label,
+  input[name="donation_amount"] + label {
+    color: #333333;
+    background: #EAEFF2;
+    transition: all .3s ease 0s;
+    &:before {
+      display: none;
+    }
     &:hover {
-      background-color: #53A0F0 !important;
+      background: #43A6E2;
+      color: #fff;
+      .credit-card-logos {
+        background-position: -4px -5px, center;
+      }
+      .paypal-logo {
+        background-position: -6px -111px,center;
+      }
+    }
+  }
+  .payment-type:checked + label,
+  input[name="donation_amount"]:checked + label {
```

Comment:
Anything that was in the old css that we now need is removed from old css and moved into here. all changes in this file cept the removal of important are this kind of change.